components/bsx/Gallery/Item/AvailableActions.vue

```diff
@@ -16,11 +16,12 @@
       :is="showMeta"
       @input="updateMeta"
       emptyOnError />
+    <DaySelect v-if="showDaySelect" v-model="selectedDay" :days="dayList" />
     <SubmitButton
       v-if="showSubmit"
       @click="submit"
       :disabled="disableSubmitButton">
-      {{ $t('nft.action.submit', [selectedAction]) }}
+      {{ $t('nft.action.submit', [selectedAction.replace('_', ' ')]) }}
```

Comment:
we could use $t(`nft.event.${selectedAction}`) or getActionButtonLabel

Reply:
replace it with `$t(nft.event.${selectedAction})`